draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -899,6 +901,8 @@ Orie Steele,
 Pieter Kasselman,
 Ryosuke Abe,
 Shawn Butterfield,
+Filip Skokan,
+Christian Bormann,
```

Comment:
@bc-pi can you please put in the alphabetic order? `Matthew Miller` seems to be out of place too.

Reply:
Will do. Sorry, I somehow missed that they were in alphabetic order on first name (guess I was looking at last).

Reply:
done w/ ca20ac44bdc0c2d9022256ad7526464d67c73620